src/plugin.js

```diff
@@ -175,12 +142,12 @@ const perSourceBehaviors = function() {
      *
      * @return {Boolean}
      */
-    disable() {
-      window.clearTimeout(srcChangeTimer);
+    disable: videojs.bind(this, function disable() {
+      this.clearTimeout(srcChangeTimer);
```

Comment:
This change is semi-unrelated, but we should've been using `Player#clearTimeout` to go along with `Player#setTimeout`.